source/making_payments/index.html.md.erb

```diff
@@ -38,7 +38,9 @@ For example:
 
 The amount must be a number data type rather than a string.
 
-The minimum amount is one pence. The maximum amount is 10,000,000 pence (£100,000). [Contact us](/support_contact_and_more_information/#contact-us) if you need to take payments above the maximum amount.
+The minimum amount is one pence plus your Payment Service Provider's (PSP's) transaction fee. Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) and check your contract to see your PSP's transaction fee.
```

Comment:
If they have our PSP, they can sign in to view the contract and check the fee. The contract is in a link in the footer.

If they have brought their own contract, they can check their transaction fees in their contract (Pay doesn't have a copy of it).

Thanks!